Patch status: fixed in >=chromium-133.0.6892.0

IWYU issue, only manifests with gcc+musl (clang+musl or gcc+glibc are fine).
Backport of [1] (>=chromium-133.0.6892.0) and [2] (>=chromium-128.0.6571.0)

Note have not tested USE=-jumbo-build which may expose more of these issues.

TODO: merge in patchset's cstdint.patch and keep a note to review for 128/133
given a lot of commits been adding cstdint/stdint.h and all hunks may not
necessarily fail to apply when fixed given headers being added differently

[1] https://chromium.googlesource.com/external/webrtc/+/000f04375b67e6139a1dc88d832d2da65c7acb29
[2] https://crrev.com/0d3d5062f7d5ff83f38685bf864c71dd5bd9d460
--- a/src/3rdparty/chromium/content/public/browser/mhtml_generation_result.h
+++ b/src/3rdparty/chromium/content/public/browser/mhtml_generation_result.h
@@ -7,2 +7,3 @@
 
+#include <cstdint>
 #include <optional>
--- a/src/3rdparty/chromium/third_party/webrtc/modules/audio_coding/neteq/reorder_optimizer.cc
+++ b/src/3rdparty/chromium/third_party/webrtc/modules/audio_coding/neteq/reorder_optimizer.cc
@@ -12,2 +12,4 @@
 
+#include <stdint.h>
+
 #include <algorithm>
